libminifi/src/core/yaml/YamlConfiguration.cpp

```diff
@@ -19,6 +19,7 @@
 #include <memory>
 #include <vector>
 #include <set>
+#include <regex>
```

Comment:
Not all versions of the compiler support <regex> You'll need to handle the C and C++ paths. I think GetFile is an example.